src/template.proto

```diff
@@ -14,6 +14,8 @@ message AnalyzeTemplate {
     string modifiedTime = 4;
     // Language code, in ISO-639 format, https://en.wikipedia.org/wiki/ISO_639-1
     string languageCode = 5;
+    // Optional flag to request all existing entities. if true, "fields" must be empty.
+    bool allEntities = 6;
```

Comment:
Inconsistent terminology.
Either change fields to entities or allEntities to allFields.
I prefer the first option.

Reply:
@limorl , I fear that renaming fields to entities will exceed the time allocated for this task.
How about to add another task to rename fields to entities, all over the project?

Reply:
We cannot have every task causing the creation of another task with development depth growing each sprint. This is a minor task, closing it correctly will probably add a few hours only. 
so please fix according to my comment in one of the two provided options.

Reply:
@limorl During the code writing I noticed some methods converting fields to entities. Therefore, for now, the name is renamed to allFields to maintain the consistency in the analyzer code.

Reply:
Can you also change the field numbering and move allFields to be the one following 'fields'?

Reply:
@itye-msft please fix following Limor's suggestion and let's merge this. (move allFields to be the one following fields)